apps/api/lib/api/client/channel.ex

```diff
@@ -1,6 +1,9 @@
 defmodule API.Client.Channel do
   use API, :channel
-  alias API.Client.Presence
+  alias Domain.Clients
+
+  # TODO: we need to self-terminate channel once the user token is set to expire, preventing
```

Comment:
Maybe, I'm just using TODOs right now to make sure that I won't miss my own tasks that I leave as I write code, because CI won't let me merge this to master unless I address them.